frontend/src/app/navbar/navbar.component.ts

```diff
@@ -11,14 +12,45 @@ import { GlobalManagerService } from '../global-manager.service';
 })
 export class NavbarComponent implements OnInit {
   loggedIn$ = this.globalManager.loggedIn$;
+  menuItems: MenuItem[];
 
   constructor(
     public loaderService: LoaderService,
     public authService: AuthService,
     private globalManager: GlobalManagerService
-  ) {}
+  ) { }
 
-  ngOnInit() {}
+  ngOnInit() {
+    this.initMenuItems();
+  }
+
+  initMenuItems() {
+    this.menuItems = [
+      {
+        label: 'Profile',
+        icon: 'pi pi-fw pi-user',
+        routerLink: ['user/profile']
+      },
+      {
+        label: 'Invite User',
+        icon: 'pi pi-fw pi-plus',
+        routerLink: ['invite']
+      },
+      {
+        label: 'Settings',
+        icon: 'pi pi-fw pi-sliders-h',
+        routerLink: ['settings']
+      },
+      {
+        separator: true
+      },
+      {
+        label: 'Logout',
+        icon: 'pi pi-fw pi-power-off',
+        command: this.logout
```

Comment:
This is throwing an error that `this.authService` is undefined.



<img width="974" alt="Screen Shot 2020-10-18 at 10 36 00 AM" src="https://user-images.githubusercontent.com/1582016/96371263-d1be2780-112e-11eb-9688-f8d8e1147c07.png">
 I believe the syntax for `command` should be

```
command: () => {
   this.authService.logout()
}
```
Furthermore, we need to make sure that the user is redirected to the login screen.